### ISSUE TITLE
Allow user to set other IP

### ISSUE THREAD

**Issue by rben01** [ritwickdey/vscode-live-server#2037]
## I'm submitting a...

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[x] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Current behavior

You have the choice of using either 127.0.0.1 or your current local IP as the address that Live Server opens to, but no other choices.

## Expected behavior

I'd like to be able to set *any* address. My local IP can change from time to time, and 127.0.0.1 is not device agnostic (so I can't copy-paste it into another device's browser).

I have a Mac and my computer's network name is `Roberts-MacBook-Pro.local` — it would be great if I could specify *this* address in Live Server's settings. This way Live Server would open to a stable URL (port notwithstanding) that I could copy-paste into other devices' browsers.

In other words, my choices for Live Server's URL are currently `127.0.0.1:5500/...` and (say) `192.168.1.50:5500/...` but I'd prefer `roberts-macbook-pro.local:5500/...`.

## Environment

```html
Browser:

- [ ] Chrome (desktop) version XX
- [ ] Firefox version XX
- [x] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: v5.6.1
- Platform:  Mac Big Sur
- Visual Studio Code: Version: 1.59.0
```

## Others

Thanks!
